usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -53,14 +57,7 @@ def post(
         ordered_json_request = json.dumps(json_request)
 
         # Check if the same request has been called today
-        # TODO!!! Use external_data_load_date to determine data freshness
-        updated_date_timestamp = datetime.strftime(datetime.now(timezone.utc), "%Y-%m-%d")
-        cached_download = (
-            DownloadJob.objects.filter(json_request=ordered_json_request, update_date__gte=updated_date_timestamp)
```

Comment:
How was this ever true? When would the last update_date ever be greater than the current time?

Reply:
`updated_date_timestamp` would have had the value of today at midnight. If a download was downloaded after midnight then the `update_date` would be greater than midnight since it is a DateTimeField.